central/deployment/queue/observation_queue.go

```diff
@@ -2,16 +2,16 @@ package queue
 
 import (
 	"container/list"
+	"time"
 
-	"github.com/gogo/protobuf/types"
 	"github.com/stackrox/rox/pkg/sync"
 )
 
 // DeploymentObservation struct used as element in the queue
 type DeploymentObservation struct {
 	DeploymentID   string
 	InObservation  bool
-	ObservationEnd *types.Timestamp
+	ObservationEnd time.Time
```

Comment:
Why not a pointer?

Reply:
all the writer sources can directly provide a time.Time object.